test/support/generator/create-party-external-rating-generator.ts

```diff
@@ -0,0 +1,74 @@
+import { ENUMS, PROPERTIES } from '@ukef/constants';
+import { DateString } from '@ukef/helpers/date-string.type';
+import { AcbsCreatePartyExternalRatingRequestDto } from '@ukef/modules/acbs/dto/acbs-create-party-external-rating-request.dto';
+import { DateStringTransformations } from '@ukef/modules/date/date-string.transformations';
+import { CreatePartyExternalRatingRequestDto } from '@ukef/modules/party-external-rating/dto/create-party-external-rating-request.dto';
+
+import { AbstractGenerator } from './abstract-generator';
+import { RandomValueGenerator } from './random-value-generator';
+
+export class CreatePartyExternalRatingGenerator extends AbstractGenerator<PartyExternalRatingValues, GenerateResult, GenerateOptions> {
+  constructor(protected readonly valueGenerator: RandomValueGenerator, protected readonly dateStringTransformations: DateStringTransformations) {
+    super(valueGenerator);
+  }
+
+  protected generateValues(): PartyExternalRatingValues {
+    const possibleAssignedRatingCodes = Object.values(ENUMS.ASSIGNED_RATING_CODES);
+
+    return {
+      assignedRatingCode: possibleAssignedRatingCodes[this.valueGenerator.integer({ min: 0, max: possibleAssignedRatingCodes.length - 1 })],
```

Comment:
You can use `this.valueGenerator.enumValue(ENUMS.ASSIGNED_RATING_CODES)` instead here :)

Reply:
Oh cool, thanks!